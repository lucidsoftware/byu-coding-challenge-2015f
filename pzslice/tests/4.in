50 50
329 825
990 258
758 937
33 229
796 540
547 30
701 984
422 524
48 934
787 965
581 258
672 680
82 835
569 331
518 775
113 168
778 558
733 528
193 934
29 920
601 695
951 445
416 434
190 588
524 819
417 346
666 182
967 677
690 232
477 935
565 828
572 177
952 237
14 971
9 31
261 64
78 236
812 361
642 28
782 55
231 345
885 49
784 577
499 176
273 218
54 595
585 813
440 594
814 474
548 523