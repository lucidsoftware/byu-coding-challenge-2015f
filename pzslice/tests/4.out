177561.499582
